third_party/move/move-compiler-v2/transactional-tests/tests/tests.rs

```diff
@@ -9,6 +9,18 @@ use std::path::Path;
 
 datatest_stable::harness!(run, TEST_DIR, r".*\.move$");
 
+/// Root of tests which require to skip v1-v2 comparison
+const SKIP_V1_COMPARISON_PATH: &str = "/no-v1-comparison/";
+
 fn run(path: &Path) -> Result<(), Box<dyn std::error::Error>> {
-    vm_test_harness::run_test_with_config(TestRunConfig::ComparisonV1V2, path)
+    let test_config: TestRunConfig = if let Some(p) = path.to_str() {
+        if p.contains(SKIP_V1_COMPARISON_PATH) {
```

Comment:
This check slightly differs from the comment above about "root of tests": we are not performing *that* check. Which might be okay, but we should say the appropriate thing in the comment above for `SKIP_V1_COMPARISON_PATH`.

Reply:
A bit nit-picky.  But ok.